src/Seq.Api/Client/SeqApiClient.cs

```diff
@@ -56,7 +58,8 @@ public class SeqApiClient : IDisposable
         /// <param name="serverUrl">The base URL of the Seq server.</param>
         /// <param name="apiKey">An API key to use when making requests to the server, if required.</param>
         /// <param name="useDefaultCredentials">Whether default credentials will be sent with HTTP requests; the default is <c>true</c>.</param>
-        public SeqApiClient(string serverUrl, string apiKey = null, bool useDefaultCredentials = true)
+        /// <param name="serverCertificateCustomValidationCallback">Callback for custom certificate validation</param>
```

Comment:
I think that appending callback to existing HttpMessageHandler is a better solution than replacing the whole  HttpMessageHandler. Because of that way no need for changing existing logic.

Reply:
I had the same thought at first, but, adding a parameter here will be a binary breaking change, which will make it harder to release, and, there are several more properties on `HttpClientHandler` that consumers may need to set in different situations in the future (`Credentials`/`DefaultProxyCredentials`/`Proxy`/`PreAuthenticate`, `SslProtocols`, `MaxConnectionsPerServer`) - which might cause further parameter list churn.